src/core/utils/stats-get-value.ts

```diff
@@ -0,0 +1,29 @@
+import BigNumber from 'bignumber.js';
+import { getTokenConfig } from '../../redux/tokens/static-selectors';
```

Comment:
pass the token config through params - we want to remove getTokenConfig from core